src/workspaces/features/UploadObjectDialog/UploadObjectDialog.tsx

```diff
@@ -74,7 +74,7 @@ const UploadObjectDialog = (props: UploadObjectDialogProps) => {
       </Dialog.Title>
       <Dialog.Content>
         <Dropzone
-          className="h-48"
+          className="h-80"
```

Comment:
Could you also enlarge the width of this dialog (`<Dialog maxWidth="max-w-3xl" ...`)